<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="button1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>391</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>从文件中读入的数据将显示在这里</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>绘制图形和交点</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>410</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>470</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; color:#550000;&quot;&gt;输入格式：L 0 0 1 1&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; color:#550000;&quot;&gt;输入完毕后，点击“添加”或“删除”按钮&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>添加/删除的数据将显示在这里</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
